src/fights.ts

```diff
@@ -416,13 +425,12 @@ export function dailyFights(): void {
             }
           }
 
-          const professorRequirement = have($item`Pocket Professor memory chip`)
-            ? new Requirement(maximizeParameters, {
-                forceEquip: $items`Pocket Professor memory chip`,
-              })
-            : new Requirement(maximizeParameters, {});
+          const profSpec = {
+            familiar: $familiar`Pocket Professor`,
+            equip: $items`Pocket Professor memory chip`.filter((i) => have(i)),
+          };
 
-          goalMaximize(Requirement.merge([professorRequirement, ...fightSource.requirements]));
+          goalMaximize({ ...profSpec, ...fightSource.spec });
```

Comment:
You could potentially run into some issues merging specs this way. For example if any of the fightSource has an `equip: Item[]` set it will just override the memory chip in the professor spec.

I should add an OutfitSpec merge to grimoire...